### ISSUE TITLE
Request value resets to default

### ISSUE THREAD

**Issue by jeksmith** [danielsomerfield/yer-song#20]
Steps to repro:

1. Select a song 
2. "touch" the $5 text box.
3. Replace 5 with 10 and touch "done"
4. Notice that the keypad goes away and that the main screen returns. $10 is there briefly, but then reverts to $5 in a couple of seconds.

I'm using Safari on an iPhone.

**Comment by danielsomerfield** [comment]
Yeah. That's a "feature". It reloads because the status could change. But that isn't going to work here as written. I just need to make sure not to reload everything and just reload the right things. Should be easy to fix.

**Comment by danielsomerfield** [comment]
Deployed.
